test/downstream/batchedinterface_arrayvars.jl

```diff
@@ -0,0 +1,42 @@
+using Symbolics
+using SymbolicIndexingInterface
+
+@variables x[1:2] y z
+
+syss = [
+    SymbolCache([x..., y]),
```

Comment:
Is splatting required? Would `[x, y]` work?

Reply:
Splatting is required. `SymbolCache` treats the variables as-is, so it would think `x` is at index 1 and `y` at 2

Reply:
Ah, I see. But for array parameters in MTK, that would be different?

Reply:
Yes. This is specifically only needed for `SymbolCache`, MTK handles it completely differently. I didn't want to pull in MTK for this one test so I used Symbolics and the rest of the SII infrastructure